pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -26,13 +26,13 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 %script{src: "/shared/js/angular-sanitize/angular-sanitize.min.js"}
 %script{src: "/shared/js/ng-csv/ng-csv.min.js"}
 
-%link(rel='stylesheet' type='text/css' href='/css/learn_tabs.css')
-%link(rel='stylesheet' type='text/css' href='/style.css')
-%link(rel='stylesheet' type='text/css' href='/css/page.css')
-%link(rel='stylesheet' type='text/css' href='/shared/css/course-blocks.css')
-%link(rel='stylesheet' type='text/css' href='/shared/css/teacher-announcement.css')
-%link(rel='stylesheet' type='text/css' href='/shared/css/teacher-dashboard.css')
-%link(rel='stylesheet' type='text/css' href='/shared/css/user-progress.css')
+%link{rel: 'stylesheet', type: 'text/css', href: '/css/learn_tabs.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/style.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/css/page.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/course-blocks.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/teacher-announcement.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/teacher-dashboard.css'}
+%link{rel: 'stylesheet', type: 'text/css', href: '/shared/css/user-progress.css'}
 
```

Comment:
I only made this change because the linter was having a fit about the hash syntax.

Reply:
Always feed the linter!